src/herbpy/action/pouring.py

```diff
@@ -0,0 +1,104 @@
+import openravepy, os, time, prpy, actions, math
+from prpy.action import ActionMethod
+from herbpy.tsr.cup import *
+from herbpy.tsr.pitcher import *
+
+def PourWater(robot, cup, pitcher):
+	"""
+	@param robot The robot performing the pouring
+	@param cup The cup to pour into
+	@param pitcher The pitcher to pour
+	"""	
+
+	manip = robot.left_arm
+    manip_cup = robot.right_arm
```

Comment:
It would be nice to make this an argument. Or does it only work on the right arm?